src/colorizer/Dataset.ts

```diff
@@ -442,8 +446,10 @@ export default class Dataset {
       }
     });
 
-    if (this.features.size === 0) {
-      throw new Error("No features found in dataset. Is the dataset manifest file valid?");
+    if (this.features.size !== manifest.features.length) {
+      throw new Error(
+        "One or more features could not be loaded. This may be because of an unsupported format or a missing file. See console for details."
+      );
```

Comment:
Behavior question: If one feature fails to load, should the whole dataset load be counted as a failure?